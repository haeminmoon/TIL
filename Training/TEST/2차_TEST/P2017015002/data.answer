Case #1
932802733
Case #2
459412330
Case #3
474859017
Case #4
387470115
Case #5
375994711
Case #6
739445869
Case #7
11920328
Case #8
165672331
Case #9
776444323
Case #10
782971285
Case #11
780934477
Case #12
374646136
Case #13
172740357
Case #14
1
Case #15
2625
Case #16
423996010
Case #17
949324146
Case #18
722114284
Case #19
77080402
Case #20
83573747
Case #21
2727101
Case #22
892437079
Case #23
140
Case #24
459105812
Case #25
68091407
Case #26
208267320
Case #27
394383
Case #28
1
Case #29
431286602
Case #30
805973457
Case #31
202997670
Case #32
83352830
Case #33
143
Case #34
445807430
Case #35
898661782
Case #36
182940422
Case #37
22249420
Case #38
659211796
Case #39
1
Case #40
889394654
Case #41
99
Case #42
968423338
Case #43
789215416
Case #44
639074738
Case #45
6201
Case #46
830993497
Case #47
293114640
Case #48
968327040
Case #49
445807859
Case #50
315778591
Case #51
16459188
Case #52
531417563
Case #53
648565699
Case #54
893509167
Case #55
633537163
Case #56
901016794
Case #57
712705126
Case #58
532456476
Case #59
121671191
Case #60
159931938
Case #61
550414921
Case #62
239847852
Case #63
847096797
Case #64
976842081
Case #65
102205752
Case #66
498738503
Case #67
42928600
Case #68
57248339
Case #69
819823975
Case #70
33644035
Case #71
163435120
Case #72
776302249
Case #73
413361946
Case #74
140
Case #75
634908458
Case #76
566139535
Case #77
10072740
Case #78
260472537
Case #79
40
Case #80
322995614
Case #81
558901503
Case #82
840693132
Case #83
5184036
Case #84
452811514
Case #85
303560230
Case #86
309654689
Case #87
949054120
Case #88
820820479
Case #89
244191386
Case #90
564873774
Case #91
248242579
Case #92
13
Case #93
158804465
Case #94
674874112
Case #95
977040343
Case #96
46210063
Case #97
638278714
Case #98
895729460
Case #99
74308148
Case #100
2731365
