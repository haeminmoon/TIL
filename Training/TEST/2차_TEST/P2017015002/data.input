100
224 60
391 83
471 85
413 31
161 72
254 87
100 94
403 64
499 36
248 50
386 94
346 71
405 62
3 2
127 3
451 82
124 10
308 9
135 65
407 40
405 4
100 24
17 13
243 98
466 13
39 26
39 32
14 13
500 85
326 69
36 16
70 13
14 9
73 31
76 21
299 56
442 40
467 93
30 1
409 75
13 5
293 97
309 64
381 69
55 51
360 54
427 97
492 75
386 6
401 23
221 38
472 85
398 16
173 69
472 50
78 63
404 41
311 87
441 14
221 23
352 48
158 33
178 40
454 20
355 23
66 43
81 7
468 47
229 85
53 41
314 17
408 39
299 13
17 13
209 87
316 89
499 49
397 95
17 14
203 28
158 100
308 21
58 7
336 84
112 10
437 14
392 53
401 11
217 32
111 95
165 78
28 2
222 10
176 94
378 40
212 88
95 57
483 74
379 46
31 20
