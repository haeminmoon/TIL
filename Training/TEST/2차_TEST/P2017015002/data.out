Case #1
5293628
Case #2
7420519
Case #3
15243377
Case #4
14597859
Case #5
15775230
Case #6
14275586
Case #7
26068452
Case #8
34027826
Case #9
34187362
Case #10
21929396
Case #11
50677056
Case #12
37054113
Case #13
65078650
Case #14
69014618
Case #15
73329804
Case #16
17426953
Case #17
54179061
Case #18
90964954
Case #19
67569062
Case #20
62132066
